src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier(random);
+        Lottery lottery = new Lottery(colorSupplier, random);
+
+        Ball ball1 = lottery.getRandomBall();
```

Comment:
Bad approach to create objects line by line. What if you have 100 objects? 
[Use for loop for creating several objects of the same class](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-for-loop-for-creating-several-objects-of-the-same-class)

Reply:
Not fixed. If you have some questions about this comment, please text in chat)